src/entities/fractions/fraction.rs

```diff
@@ -238,107 +255,122 @@ mod tests {
 
     #[test]
     fn test_quotient() {
-        assert_eq!(Fraction::new(8, 3).quotient(), BigInt::from(2));
-        assert_eq!(Fraction::new(12, 4).quotient(), BigInt::from(3));
-        assert_eq!(Fraction::new(16, 5).quotient(), BigInt::from(3));
+        assert_eq!(Fraction::new(8, 3).unwrap().quotient(), BigInt::from(2));
+        assert_eq!(Fraction::new(12, 4).unwrap().quotient(), BigInt::from(3));
+        assert_eq!(Fraction::new(16, 5).unwrap().quotient(), BigInt::from(3));
     }
 
     #[test]
     fn test_remainder() {
-        assert_eq!(Fraction::new(8, 3).remainder(), Fraction::new(2, 3));
-        assert_eq!(Fraction::new(12, 4).remainder(), Fraction::new(0, 4));
-        assert_eq!(Fraction::new(16, 5).remainder(), Fraction::new(1, 5));
+        assert_eq!(
+            Fraction::new(8, 3).unwrap().remainder().unwrap(),
+            Fraction::new(2, 3).unwrap()
+        );
+        assert_eq!(
+            Fraction::new(12, 4).unwrap().remainder().unwrap(),
+            Fraction::new(0, 4).unwrap()
+        );
+        assert_eq!(
+            Fraction::new(16, 5).unwrap().remainder().unwrap(),
+            Fraction::new(1, 5).unwrap()
+        );
```

Comment:
```suggestion
        assert_eq!(Fraction::new(8, 3)?.remainder()?, Fraction::new(2, 3)?);
        assert_eq!(Fraction::new(12, 4)?.remainder()?, Fraction::new(0, 4)?);
        assert_eq!(Fraction::new(16, 5)?.remainder()?, Fraction::new(1, 5)?);
        Ok(())
```
Prefer not to spam `unwrap()`.

Reply:
it's a test though, it has constants as it parameters

Reply:
I don't get it. I was saying the code can be more concise.